specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/nppBackup/privateEndpointConnections.json.2020-10-28_153436.bak

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
Remove this file?

Reply:
Good catch! Thank you for pointing this out. I have removed it.